pkgs/applications/graphics/screentest/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoconf
+, automake
+, intltool
+, libtool
+, pkg-config
+, gtk2
+}:
+
+stdenv.mkDerivation rec {
+  pname = "screentest";
+  version = "unstable-2021-05-10";
+
+  src = fetchFromGitHub {
+    owner = "TobiX";
+    repo = "screentest";
+    rev = "780e6cbbbbd6ba93e246e7747fe593b40c4e2747";
+    hash = "sha256-TJ47c77vQ/aRBJ2uEiFLuAR4dd4CMEo+iAAx0HCFbmA=";
+  };
+
+  preConfigure = "./autogen.sh";
+
+  nativeBuildInputs = [
+    autoconf
+    automake
+    intltool
+    libtool
+    pkg-config
+  ];
+
+  buildInputs = [ gtk2 ];
+
+  meta = with lib; {
+    description = "A simple screen testing tool";
+    homepage = "https://github.com/TobiX/screentest";
+    changelog = "https://github.com/TobiX/screentest/blob/${src.rev}/NEWS";
+    license = licenses.gpl2Only;
+    maintainers = with maintainers; [ evils ];
```

Comment:
```suggestion
  meta = {
    description = "A simple screen testing tool";
    homepage = "https://github.com/TobiX/screentest";
    changelog = "https://github.com/TobiX/screentest/blob/${finalAttrs.rev}/NEWS";
    license = lib.licenses.gpl2Only;
    maintainers = with lib.maintainers; [ evils ];
```

Reply:
that doesn't really matter except the finalAttrs part